_data/projects/New-Programming-Languages-Showcase.yml

```
@@ -0,0 +1,16 @@
+name: New-Programming-Languages-Showcase
+desc: This repository is a collection of information and resources on various new programming languages. Here, you can find descriptions, development details, advantages, usage examples, successful applications, and future prospects of these languages. We hope that this repository will be a helpful resource for developers looking to explore new programming languages.
+site: https://github.com/multikitty/New-Programming-Languages-Showcase
+tags:
+  - programming-language
+  - new-language
+  - showcase
+  - brief-history
+  - development-details
+  - future-technologies
+  - supporter
+  - advantages
+  - news
+upforgrabs:
+  name: Wecome, contributors!
+  link: https://github.com/multikitty/New-Programming-Languages-Showcase/issues/1
```

This should be a URL to a label in the repository, matching the `name` value defined above.

Looking at the contents of that issue I think either `help wanted` or `good first issue` are what you could use here:

```suggestion
  name: help wanted
  link: https://github.com/multikitty/New-Programming-Languages-Showcase/labels/help%20wanted
```

---

Thanks for your suggestion.